posts/bonus-AsyncLocalStorage.md

```diff
@@ -0,0 +1,186 @@
+# Managing Request-Scoped Data in Fastify
+
+I was looking at the [Fastify plugin list](https://eomm.notion.site/7a064537ee794af698684df68e215b54?v=4034009f43bd4d599a31701c4246d9fa&pvs=4)
+and found myself asking, _"What is this [`@fastify/request-context` plugin](https://github.com/fastify/fastify-request-context)?"_  
+So, I checked it out, and here I am, writing a post about it to share what it is and how it can be useful for you!
+
+## What is AsyncLocalStorage?
```

Comment:
is it experimental?

Reply:
no

<img width="621" alt="image" src="https://github.com/user-attachments/assets/4be469e3-77d5-4b68-9a6a-8f35f5f222ef" />